src/otx/algo/common/utils/bbox_overlaps.py

```diff
@@ -142,15 +142,13 @@ def bbox_overlaps(
         >>> assert tuple(bbox_overlaps(nonempty, empty).shape) == (1, 0)
         >>> assert tuple(bbox_overlaps(empty, empty).shape) == (0, 0)
     """
-    batch_shape = bboxes1.shape[:-2]
-
     rows = bboxes1.size(-2)
     cols = bboxes2.size(-2)
 
     if rows * cols == 0:
         if is_aligned:
-            return bboxes1.new((*batch_shape, rows))
-        return bboxes1.new((*batch_shape, rows, cols))
+            return bboxes1.new((*bboxes1.shape[:-2], rows))
+        return bboxes1.new((*bboxes1.shape[:-2], rows, cols))
```

Comment:
FYI. It can overlap with this change targeting 2.2.0. Check here when merging back.
https://github.com/openvinotoolkit/training_extensions/pull/4010/files#diff-2c816c2342cac19b9d44901971ec8d8e0189c0eb05e084764066ad170144b0bdR163-R164

Reply:
thanks, fixed - https://github.com/openvinotoolkit/training_extensions/pull/4003/commits/d9787fdfe8f544b5f317ddd292bf4beaaf244c0b